src/DigitalSubscriberAppBanner/canRender.ts

```diff
@@ -0,0 +1,10 @@
+import type { BrazeMessageProps } from './index';
+export const COMPONENT_NAME = 'DigitalSubscriberAppBanner';
+
+export const canRender = (brazeMessageProps: BrazeMessageProps): boolean => {
+    const { header, body } = brazeMessageProps;
+    if (!header || !body) {
+        return false;
+    }
+    return true;
```

Comment:
No strong preference, but we could write this as:

```suggestion
    return Boolean(header && body);
```

Reply:
Done!